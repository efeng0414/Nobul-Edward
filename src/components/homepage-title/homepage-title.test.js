import React from "react";
import renderer from "react-test-renderer";
import HomepageTitle from "./index";

describe("HomepageTitle", () => {
  const componentProps = {
    title: "homepage"
  };
  const componentJSX = <HomepageTitle {...componentProps} />;
  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
