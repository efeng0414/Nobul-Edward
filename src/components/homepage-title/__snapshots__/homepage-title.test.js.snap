// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageTitle Component Snapshot should match existing snapshot 1`] = `
<h1
  className="homepage-title"
>
  homepage
</h1>
`;
